### PR TITLE
add custom fonts

### PR DESCRIPTION
**Pull request by elramus**
Could be wrong on this, but from what I can tell from research, the best way to serve (self-hosting vs google cdn) is kind of a mixed bag. Speed of self-hosting depends on our own servers, and whether or not they're not as fast, caching optimally, etc. Google's are very fast, so self-hosting could actually be slower. Apparently the big advantage of self-hosting is that you can play the actual font file yourself and remove characters you won't need... not sure if we want to go there or not.

So, I added a DNS prefetch and kept it loading from google's CDNs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,5 @@
 
 # Ignore Byebug command history file.
 .byebug_history
+
+.DS_Store
```
Comment (tycobbb) [inline .gitignore]: i recommend adding this to a global gitignore on your machine instead.

https://help.github.com/articles/ignoring-files/

File: app/javascript/src/shared/styles/mixins/fonts.js [hunk 2]
```diff
@@ -4,15 +4,26 @@ type FontFace = {|
...
-const fontFamily = 'Nunito, sans-serif'
+const fontFamilies = {
+  serif:'Zilla Slab, serif',
+  sans:'Roboto, sans-serif'
+}
 
-export const fonts: { regular: FontFace, bold: FontFace } = {
-  regular: {
-    fontFamily,
+export const fonts: { lightSans: FontFace, regularSans:FontFace, boldSans: FontFace, regularSerif: FontFace } = {
```
Comment (tycobbb) [inline app/javascript/src/shared/styles/mixins/fonts.js]: what do you think about naming these in the format:

`light` for sans styles
`lightSlab` for slab styles

to keep naming terse for the common case

**Review by tycobbb — CHANGES_REQUESTED** [review]
Can you make sure you're passing the linter / tests by running `yarn validate`?

File: app/javascript/src/shared/styles/mixins/fonts.js (hunk 2 above)
Comment (elramus) [reply app/javascript/src/shared/styles/mixins/fonts.js]: sure that sounds good.

**Review by tycobbb — APPROVED** [review]
LGTM.

**Comment by elramus** [comment]
Ok should be good to go now. Sorry about all the messy merging :P

File: app/javascript/src/shared/styles/fontfaces.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import robotoRegularWOFF from '../../../fonts/roboto-regular.woff'
+import robotoRegularTTF from '../../../fonts/roboto-regular.ttf'
+
+import robotoBoldWOFF2 from '../../../fonts/roboto-bold.woff2'
+import robotoBoldWOFF from '../../../fonts/roboto-bold.woff'
+import robotoBoldTTF from '../../../fonts/roboto-bold.ttf'
+
+import zillaSlabRegularWOFF2 from '../../../fonts/zillaslab-regular.woff2'
+import zillaSlabRegularWOFF from '../../../fonts/zillaslab-regular.woff'
+import zillaSlabRegularTTF from '../../../fonts/zillaslab-regular.ttf'
```
Comment (tycobbb) [inline app/javascript/src/shared/styles/fontfaces.js]: i don't think we need ttf if we have woff/2, do you think we can get rid of it?

File: app/javascript/src/shared/styles/fontfaces.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+// @flow
+import { css } from 'glamor'
+import robotoLightWOFF2 from '../../../fonts/roboto-light.woff2'
```
Comment (tycobbb) [inline app/javascript/src/shared/styles/fontfaces.js]: can we camel-case these names, e.g. `robotoLightWoff2`? also, was there a problem with exporting all of these fonts from a shared index to clean up this import? i can't remember.

File: db/seeds.rb [hunk 5]
```diff
@@ -1,3 +1,4 @@
+require './spec/helpers/factory_helpers'
```
Comment (tycobbb) [inline db/seeds.rb]: i think you may have merged this file backwards. we want to delete this line and create the hearings for different weeks below.

File: spec/factories/hearing_factory.rb [hunk 6]
```diff
@@ -13,13 +13,5 @@
     trait :with_committee do
       committee
     end
-
-    trait :this_week do
```
Comment (tycobbb) [inline spec/factories/hearing_factory.rb]: same here, let's not delete these.

**Review by tycobbb — CHANGES_REQUESTED** [review]

File: app/javascript/src/shared/styles/fontfaces.js (hunk 4 above)
Comment (elramus) [reply app/javascript/src/shared/styles/fontfaces.js]: Aaah yes you're right. We set that up and then while I was debugging something else I removed that. Will switch back, and fix camel-case.

File: app/javascript/src/shared/styles/fontfaces.js (hunk 3 above)
Comment (elramus) [reply app/javascript/src/shared/styles/fontfaces.js]: Sure

File: db/seeds.rb (hunk 5 above)
Comment (elramus) [reply db/seeds.rb]: yeah there's no need for any changes to this file in this PR, will remove it

File: spec/factories/hearing_factory.rb (hunk 6 above)
Comment (elramus) [reply spec/factories/hearing_factory.rb]: yup

**Review by tycobbb — APPROVED** [review]

**Pull request merged**